server/startup/migrations/v238.ts

```diff
@@ -0,0 +1,34 @@
+import { Mongo } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations/server';
+import { isEnterprise } from '../../../ee/app/license/server';
+import { Settings } from '../../../app/models/server/raw';
+// import { Banner } from '../../sdk';
+// import { BannerPlatform } from '../../../definition/IBanner';
+// import { IUser } from '../../../definition/IUser';
+import { settings } from '../../../app/settings/server';
+
+Migrations.add({
+	version: 238,
+	up() {
+		const query = {
+			_id: { $in: [/^Accounts_OAuth_Custom-?([^-_]+)$/] },
+			value: true,
+		};
+
+		const isCustomOAuthEnabled = !!Promise.await(Settings.findOne(query));
+		const LDAPEnabled = settings.get('LDAP_Enable');
+		const SAMLEnabled = settings.get('SAML_Custom_Default');
+
+		const isEE = isEnterprise();
+
+		if (!isEE && (isCustomOAuthEnabled || LDAPEnabled || SAMLEnabled)) {
```

Comment:
Could check the banner collections if cloud already sent the alert earlier to not show it again

Reply:
I don't think this will be necessary. Since I'm just updating the text field from these documents, everything else should work the same. In other words, this banner won't show up for people who clicked on the dismiss button.